web/src/main/java/com/synopsys/integration/alert/web/documentation/SwaggerConfiguration.java

```diff
@@ -56,6 +58,14 @@ public Docket api() {
                    .apiInfo(apiEndpointInfo());
     }
 
+    @Bean
+    public UiConfiguration alertSwaggerUiConfiguration() {
+        return UiConfigurationBuilder
+                   .builder()
+                   .supportedSubmitMethods(new String[] { "get" })
```

Comment:
This only works if it is lower-case.

Reply:
Can we turn it into a constant with a comment above it stating this fact?